src/states.js

```diff
@@ -1,29 +1,25 @@
-/*
- * This file is necessary to avoid this rollup issue:
- * https://github.com/rollup/rollup/issues/1089
```

Comment:
Was this a different circular dependency than this issue?

Reply:
Yeah, this issues was only in rollup 0.50.1, after that another issue pops up. That was the one that was fixed in this pull request. rollup would still build the files correctly but would append `$$1` to most of the states to avoid the circular dependency problem. Since most of the tests use the constructor name this caused them to fail.